docs/topics/con_what-is-springboot.adoc

```diff
@@ -1,7 +1,7 @@
 [id='what-is-springboot_{context}']
 = What is {SpringBoot}
 
-{SpringBoot} lets you create opinionated Spring-based stand-alone applications.
+{SpringBoot} lets you create prescriptive Spring-based stand-alone applications.
```

Comment:
While "opinionated applications" sounds weird, it does convey some meaning to me. "Prescriptive applications" -- not really.

Reply:
Do you think I could just leave that word out?

Reply:
Totally.